androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/PlayerControls.kt

```diff
@@ -108,10 +132,13 @@ private fun PreviewPlayerControls() {
         isPlaying = true,
         hasEnded = false,
         isFullScreen = false,
+        speed = 1f,
```

Comment:
nit: create a const default value and replace instead of plain value `1f`

Reply:
I'm assuming we have to define a default of the speed list somewhere else, so we can reuse it here

Reply:
Considering that this is just the view, I think it's ok to use a plain value here, the value that I use in the component is taken directly from the exoplayer, so I don't need to set a default speed there